agent/bench.go

```diff
@@ -549,7 +562,20 @@ func (b *Bench) RerunKube(cmd, cmdRemap string, forced bool) {
 	}
 
 	_, b.isKubeMaster = b.kubeCisCmds[cmdKubeApiServer]
+	if !b.isKubeMaster {
+		if _, b.isKubeMaster = b.kubeCisCmds[cmdK3sServer]; b.isKubeMaster {
```

Comment:
What is the command in your k3s environment?  I installed my instance following [the quick start guide](https://docs.k3s.io/quick-start), and my command line looks like this:

```
root        2061       1 10 09:22 ?        00:29:13 /usr/local/bin/k3s agent
```

```
$ /usr/local/bin/k3s -v
k3s version v1.29.6+k3s2 (b4b156d9)
go version go1.21.11
```

Not sure if it would be supported in your case.

Reply:
Hi @holyspectral ,
I add `isKubeProcess` to detect the executable of k3s not proc name, since the proc name could change over time, and I keep the original logic for backward compatible.